scikits/umfpack/setup.py

```diff
@@ -28,6 +28,13 @@ def umfpack_i(ext, build_dir):
     build_info = {}
     dict_append(build_info, **umf_info)
     dict_append(build_info, **blas_info)
+    
+    # On windows, vcpkg names the library 'libumfpack'
+    # rather than 'umfpack' and this change is not yet
+    # in numpy; therefore, we fix the problem here
+    if sys.platform == 'win32':
+        build_info['libraries'].remove('umfpack')
+        build_info['libraries'].append('libumfpack')
```

Comment:
For the record, I don't understand why this is required. https://github.com/numpy/numpy/blob/e64699dcaca9eb0dd97deabae01ffc2884cacbb0/numpy/distutils/system_info.py#L741-L757 should have found the correct library.